media_driver/linux/common/codec/ddi/media_libvpx_vp9.cpp

```diff
@@ -97,9 +97,9 @@ void write_bitdepth_colorspace_sampling(uint32_t codecProfile,
         (codecProfile == VP9_PROFILE_3))
     {
         /* sub_sampling_x/y */
-        /* Currently the sub_sampling_x = 0, sub_sampling_y = 1 */
+        /* Currently the sub_sampling_x = 0, sub_sampling_y = 0 */
```

Comment:
PROFILE1 and profile3, suppose should cover both 444 440 420?

Reply:
Only PROFILE 0 / PRFOFILE 2 is cover for 420.  PROFILE 1  and PROFILE 3 is cover both 440 and 444. This information are coming from vp9-bitstream-specification-v0.6-20160331-draft.pdf under section 7.2 Uncompressed header semantics.  Does the media driver side got support for VP9 encode YUV440?

Reply:
accept it, because we does not support 440, so , always is 444.